plugins/ssr/ssrelim.ml

```diff
@@ -485,16 +485,15 @@ let revtoptac n0 =
   Logic.refiner ~check:true EConstr.Unsafe.(to_constr (EConstr.mkApp (f, [|Evarutil.mk_new_meta ()|])))
   end
 
+let warn_injection =
+   CWarnings.create ~name:"nothing-to-inject" ~category:"ssr"
+     (fun x -> Pp.str x)
```

Comment:
It would make more sense to put `fun () -> Pp.str "Nothing to inject"` here. It is not the responsibility of the caller to define what the warning says.

Also, the convention in the rest of the code is to use the warning name, i.e. `warn_nothing_to_inject`.

Reply:
ok